unicef/etools
# move source code under 'etools' package

- move all django applications under 'etools.applications' namespace
- move all libraries/modules under 'etools.libraries' namespace
- move project configuration/ settings under 'etools.config' namespace
- move 'etools' package under src directory
- move all scripts to root folder (manage.py,manage_debug.sh)
REQUIREMENTS:
- move requirements in src/requirements directory
- add isort/pdbpp/ipython to test.in requirements
TESTS/CIRCLECI
- updates circleci to use tox and produce html coverage reports
- updates tox.ini and django 1.11/2.0 matrix
- updates .coveragerc
CODE
- order all imports following iSort rules
- removes all 'from __future__' imports
- fixes settings SITE_NAME and ROOT_URLCONF
- set MEDIA_ROOT and STATIC_ROOT to tmp/etools
- update manage.py to be executed in the root folder
- creates setup.py
- creates MANIFEST.in
- updates runtests.sh to execute flake and django checks before tests
  (check of migrations is now implemented using django check framework)
DOCKER
- updates docker related files
- add default value to 'ARG REQUIREMENTS_FILE' in Dockerfile-dev
OTHERS
- silence 'tenant_schemas.W003' warning
- updates .gitignore
- change README.md to README.rst
- move isort config to .isort.cfg from setup.cfg
- add comments to flake8
